assets/src/editor/events/tickets/editor-ticket/list-view/editor-ticket-entities-list-view.js

```diff
@@ -105,4 +106,15 @@ const EditorTicketEntitiesListView = ( {
 	);
 };
 
+EditorTicketEntitiesListView.propTypes = {
+	entities: PropTypes.array.isRequired,
+	displayTicketDate: PropTypes.string,
+	htmlClass: PropTypes.string,
+	doRefresh: PropTypes.func,
+};
+
+EditorTicketEntitiesListView.defaultProps = {
+	showDesc: false,
```

Comment:
oops more copypasta... defaults should at least be present in the `propTypes ` object above, so always good to double check there.